packages/sources/react/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "noFallthroughCasesInSwitch": true,
     "rootDir": ".",
     "declaration": true,
-    "sourceMap": true,
+    "declarationDir": "dist",
```

Comment:
@Shyrro i'm a bit curious on this change, is this more "performant" to precise that the declarations are in the dist?

Reply:
@thibault-mahe It is not more performant, but rather to insure that the declarations will always go to the dist, by default they're put in the working directory, so i'd rather limit the confusion